docs/application/dotnet/guides/nui/nui-components/Tab.md

```diff
@@ -7,6 +7,17 @@ You can handle a tab by adding, inserting, or deleting a TabItem. A tab can cont
 
 ![Tab](./media/tab.png) ![Tab](./media/tab2.png)
 
+> [!NOTE]
+> Tab is deprecated in API8 (Tizen 6.0) and will be removed in API10.

```

Comment:
Please rephrase to:

Tab is deprecated since Tizen 6.0 and will be removed after two releases.